portal/code/cx-portal-shared-components/src/components/content/AppCards/AppCard.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box, useTheme } from '@mui/material'
+import { useEffect, useState } from 'react'
+import { AppCardButtons, AppCardButtonsProps } from './AppCardButtons'
+import { AppCardContent, AppCardContentProps } from './AppCardContent'
+import { AppCardImage, AppCardImageProps } from './AppCardImage'
+
+type Variants = 'minimal' | 'compact' | 'expanded' | 'preview'
+
+export interface AppCardProps
+  extends AppCardContentProps,
+    AppCardButtonsProps,
+    AppCardImageProps {
+  variant?: Exclude<Variants, 'preview'>
+}
+
+export const AppCard = ({
+  variant: variantProp = 'minimal',
+  title,
```

Comment:
instead of passing all parameters one by one we want one prop 'app' where we pass one data object to the app card:

{
  title: 'Digital Twin Aspect Debugger',
  subtitle: 'Catena-X',
  rating: 4.5,
  price: 'free to use',
  description: 'Lorem Ipsum is simply dummy text of the printing.',
  image: {
    src: '/path/to/image.jpg’,
    alt: 'Catena-X AppCard',
  },
}